semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -1351,16 +1372,16 @@ and nullable_type (env : env) ((v1, v2) : CST.nullable_type) =
    | [] -> raise Impossible (* see repeat1($._quest) in grammar.js *)
   )
 
-and parameter (env : env) ((v1, v2, v3) : CST.parameter) : parameter =
+and parameter (env : env) ((v1, v2, v3) : CST.parameter) =
   let v1 = simple_identifier env v1 in
   let v2 = token env v2 (* ":" *) in
   let v3 = type_ env v3 in
-  let param = { (param_of_id v1) with ptype = Some v3 } in
-  ParamClassic param
+  v1, v3
 
 and parameter_modifiers (env : env) (x : CST.parameter_modifiers) =
   (match x with
-   | `Anno x -> annotation env x
+   | `Anno x -> let _ = annotation env x in
+       raise Todo
```

Comment:
why Todo? You can populate the pattrs field of parameter_classic no?

Reply:
ah yes, I'll fix this